### PR TITLE
OMIM and ORDO sync

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
- reclassify 'COVID-19–associated multisystem inflammatory syndrome in adults' and 'COVID-19–associated multisystem inflammatory syndrome in children' as child of 'multisystem inflammatory syndrome in children and adults'
- write up documentation about how to do OMIM and ORDO slurp

**Notes:**
- I checked OMIM for new genetic diseases from Orphanet to see if a class already existed in OMIM and if it didn't, I added the new Orphanet class to Mondo, otherwise I just added Orphanet xrefs to existing Mondo classes. For example:
  - there is not a class in [OMIM](https://omim.org/phenotypicSeries/PS266600) that is a TRIM22-related inflammatory bowel disease
  - there is not a class in [OMIM](https://omim.org/phenotypicSeries/PS266600) that is a ALPI-related inflammatory bowel disease

**Questions:**

- [x] there is this term (see below) in Orphanet, which might be equivalent to MONDO_0035364 'oculocutaneous albinism type 8' (https://omim.org/entry/619165), but Orphanet doesn't have much info about this term, so I can't be sure. Should I add the Orphanet ID as an equiv xref on MONDO_0035364?

[Term]
id: MONDO:0035364
name: oculocutaneous albinism type 8
subset: ordo_disease
synonym: "OCA8" EXACT  [Orphanet:597733]
is_a: MONDO:0000001 {source="Orphanet:597733"} ! disease or disorder
is_a: MONDO:0018910 {source="Orphanet:597733"} ! oculocutaneous albinism
xref: Orphanet:597733 {source="MONDO:equivalentTo"} ! oculocutaneous albinism type 8

- [x] I think the term below is equivalent to this [OMIM term](https://omim.org/entry/618367) (MONDO_0032705 Neurodevelopmental disorder with microcephaly, epilepsy, and hypomyelination), but there is not a mapping b/w these two terms in OMIM or Orphanet

[Term]
id: MONDO:0035369
name: mthfs-related developmental delay-microcephaly-short stature-epilepsy syndrome
subset: ordo_disease
is_a: MONDO:0015162 {source="Orphanet:597874"} ! obsolete rare syndromic intellectual disability
is_a: MONDO:0015983 {source="Orphanet:597874"} ! obsolete rare genetic syndromic intellectual disability
is_a: MONDO:0016406 {source="Orphanet:597874"} ! obsolete other metabolic disease with epilepsy
is_a: MONDO:0017313 {source="Orphanet:597874"} ! disorder of folate metabolism and transport
is_a: MONDO:0000001 {source="Orphanet:597874"} ! disease or disorder
is_a: MONDO:0019046 {source="Orphanet:597874"} ! leukodystrophy
is_a: MONDO:0019058 {source="Orphanet:597874"} ! neurometabolic disease
xref: Orphanet:597874 {source="MONDO:equivalentTo"} ! mthfs-related developmental delay-microcephaly-short stature-epilepsy syndrome

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -422565,6 +422587,86 @@ is_a: MONDO:0002558 {source="Orphanet:590539"} ! melanotic neurilemmoma
...
+name: localized dystrophic epidermolysis bullosa
+def: "A localized form of dystrophic epidermolysis bullosa characterized by blisters confined primarily to the hands and feet (acral form) or to the pretibial region (pretibial form). Nail dystrophy or loss is common and may be an isolated finding (nail only form). This disease can be inherited via autosomal dominant or autosomal recessive inheritance." [Orphanet:595356]
+subset: ordo_disease
+synonym: "localized DEB" EXACT [Orphanet:595356]
+xref: Orphanet:595356 {source="MONDO:equivalentTo"}
+is_a: MONDO:0017608 {source="Orphanet:595356"} ! dystrophic epidermolysis bullosa
+
+[Term]
+id: MONDO:0035354
+name: IgG4-related systemic disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:422601]: what is our strategy for ensuring complete classification?

consider obsoleting on merge

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -422565,6 +422587,86 @@ is_a: MONDO:0002558 {source="Orphanet:590539"} ! melanotic neurilemmoma
...
+name: portosinusoidal vascular disease
+subset: ordo_disease
+synonym: "PSVD" EXACT ABBREVIATION [Orphanet:596937]
+xref: ICD10:K76.8 {source="Orphanet:596937/ntbt", source="Orphanet:596937"}
+xref: Orphanet:596937 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002405 {source="Orphanet:596937"} ! hepatic vascular disease
+
+[Term]
+id: MONDO:0035362
+name: TRIM22-related inflammatory bowel disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:422619]: consider obsoleting on merge

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -422565,6 +422587,86 @@ is_a: MONDO:0002558 {source="Orphanet:590539"} ! melanotic neurilemmoma
...
+subset: ordo_disease
+synonym: "TRIM22-related IBD" EXACT [Orphanet:597201]
+xref: Orphanet:597201 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005265 {source="Orphanet:597201"} ! inflammatory bowel disease
+
+[Term]
+id: MONDO:0035363
+name: IRF2BPL-related regressive neurodevelopmental disorder-dystonia-seizures syndrome
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:422627]: how is this different from?

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -422565,6 +422587,86 @@ is_a: MONDO:0002558 {source="Orphanet:590539"} ! melanotic neurilemmoma
...
+subset: ordo_disease
+xref: Orphanet:597623 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002254 {source="https://orcid.org/0000-0001-5208-3432"} ! syndromic disease
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/14282 {source="Orphanet:597623"}
+
+[Term]
+id: MONDO:0035369
+name: MTHFS-related developmental delay-microcephaly-short stature-epilepsy syndrome
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:422635]: how is this different from https://www.omim.org/entry/618367?

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -422565,6 +422587,86 @@ is_a: MONDO:0002558 {source="Orphanet:590539"} ! melanotic neurilemmoma
...
+subset: ordo_disease
+xref: Orphanet:597874 {source="MONDO:equivalentTo"}
+is_a: MONDO:0000001 {source="Orphanet:597874"} ! disease or disorder
+is_a: MONDO:0017313 {source="Orphanet:597874"} ! disorder of folate metabolism and transport
+is_a: MONDO:0019046 {source="Orphanet:597874"} ! leukodystrophy
+is_a: MONDO:0019058 {source="Orphanet:597874"} ! neurometabolic disease
+
+[Term]
+id: MONDO:0035370
+name: ALPI-related inflammatory bowel disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:422645]: is this mendelian? if so should be in omim

if not what is this? susceptibility? is it really a subtype of IBD?

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -422565,6 +422587,86 @@ is_a: MONDO:0002558 {source="Orphanet:590539"} ! melanotic neurilemmoma
...
+subset: ordo_disease
+xref: Orphanet:597887 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005265 {source="https://orcid.org/0000-0001-5208-3432"} ! inflammatory bowel disease
+
+[Term]
+id: MONDO:0035375
+name: multisystem inflammatory syndrome in children and adults
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:422652]: how is this different from

id: MONDO:0100163
name: COVID-19–associated multisystem inflammatory syndrome in children

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -422565,6 +422587,86 @@ is_a: MONDO:0002558 {source="Orphanet:590539"} ! melanotic neurilemmoma
...
+
+[Term]
+id: MONDO:0035398
+name: hypomyelination of early myelinating structures
+subset: ordo_disease
+synonym: "HEMS" EXACT ABBREVIATION [Orphanet:599376]
+xref: Orphanet:599376 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019046 {source="Orphanet:599376"} ! leukodystrophy
+is_a: MONDO:0100309 {source="Orphanet:599376"} ! hereditary ataxia
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/9086 {source="Orphanet:599376"} ! PLP1
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:422668]: remember always lookup the gene in omim

https://www.omim.org/entry/300401?search=PLP1&highlight=plp1

is it one of these?

**Review by cmungall — APPROVED** [review]

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:422601]: I created a ticket here: https://github.com/monarch-initiative/mondo/issues/3539

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:422619]: https://github.com/monarch-initiative/mondo/issues/3540

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:422627]: https://github.com/monarch-initiative/mondo/issues/3541

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:422635]: https://github.com/monarch-initiative/mondo/issues/3542

File: src/ontology/mondo-edit.obo (hunk 5 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:422645]: https://github.com/monarch-initiative/mondo/issues/3543

File: src/ontology/mondo-edit.obo (hunk 6 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:422652]: it is a grouping class, includes children and adults

File: src/ontology/mondo-edit.obo (hunk 7 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:422668]: I did look at both of those and they don't seem equivalent. I'll double-check this with Ada though.

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:422668]: https://github.com/monarch-initiative/mondo/issues/3544

**Pull request merged**